docs/common-coreboot-docs/checkbox.md

```diff
@@ -0,0 +1,222 @@
+    │[X] + Optical Drive tests                                                     │
+    │[X] + Power Management tests                                                  │
+    │[X] + Suspend tests                                                           │
+    │[X] + TPM 2.0 (Trusted Platform Module)                                       │
+    │[X] + Touchpad tests                                                          │
+            .
+            .
+            . etc.
+    └──────────────────────────────────────────────────────────────────────────────┘
+    ```
+
+1. Press `T` to start the testing procedure. You can be asked about hardware in
+your machine. Just tick it truthfully and press `T` again.
+
+    ```bash
+    ┌──────────────────────────────────────────────────────────────────────────────┐
+    │Does this machine have this piece of hardware?                                │
+    │    An Ethernet Port                              (X) YES  ( ) NO             │
+    │    Camera/Capture Device                         (X) YES  ( ) NO             │
+    │    USB Storage Device Connected                  (X) YES  ( ) NO             │
+    └──────────────────────────────────────────────────────────────────────────────┘
+    ```
+
+1. When testing end, menu with test results should appear. Press `R` to rerun
+test cases, or `F` to finish.
+
+1. After all, paths to the tests results should be displayed in terminal.
+By default, they are placed in `/home/user/.local/share/checkbox-ng`.
+
+## Additional options
+
+1. `run` lets you run particular test plan or a set of jobs, but it not save any
+   results. Example of run one test plan:
+
+    ```bash
+    sudo checkbox-cli run com.canonical.certification::smoke
+    ```
+
+2. `launcher` command lets you customize checkbox experience. To use it, you
+   need to create `config_file.ini` and start run checkbox like bellow:
+
+    ```bash
+    checkbox-cli launcher config_file.ini
+    ```
+
+For more details about this commands and also other visit
+[checkboc-cli](https://checkbox.readthedocs.io/en/latest/using.html#).
+
+## Troubleshooting
+
+When somehow checkbox will stop work you can resume previous session.
+To do this open terminal and run following command:
+
+```bash
+sudo checkbox-cli
+```
+
+If at least on machine is 1 incomplete session:
+
+```bash
+ Do you want to resume session 'session_title-2022-07-06T13.09.22'?
+  r => resume this session
+  n => next session
+  c => create new session
+  d => delete old sessions
+[rncd]:
+```
+
+Type in `r` to resume stopped session. You can also create new session or
+delete old session by typing the appropriate letter.
+
+```bash
+What do you want to do with that job?
+  s => skip that job
+  p => mark it as passed and continue
+  f => mark it as failed and continue
+  r => run it
+[spfr]:
+```
+
+Decide what do you want with the last test and type in the appropriate letter.
+After this your checkbox session will be resumed.
```

Comment:
I would ask you, after making corrections, to run the file through a validation program

Reply:
You mean pre-commit?

Reply:
In this case I mean some tool like `grammarly` to check the correctness of the grammar side of the documentation.

Reply:
I used grammarly